python/lsst/afw/math/statistics.cc

```diff
@@ -86,7 +86,16 @@ PYBIND11_PLUGIN(_statistics) {
 
     py::class_<StatisticsControl, std::shared_ptr<StatisticsControl>> clsStatisticsControl(mod, "StatisticsControl");
 
-    clsStatisticsControl.def(py::init<>());
+    py::enum_<StatisticsControl::WeightsBoolean>(clsStatisticsControl, "WeightsBoolean")
+        .value("WEIGHTS_FALSE", StatisticsControl::WeightsBoolean::WEIGHTS_FALSE)
+        .value("WEIGHTS_TRUE", StatisticsControl::WeightsBoolean::WEIGHTS_TRUE)
+        .value("WEIGHTS_NONE", StatisticsControl::WeightsBoolean::WEIGHTS_NONE)
+        .export_values();
+
+    clsStatisticsControl.def(py::init<double, int, lsst::afw::image::MaskPixel, bool,
+                                      typename StatisticsControl::WeightsBoolean>(),
+                             "numSigmaClip"_a = 3.0, "numIter"_a = 3, "andMask"_a = 0x0, "isNanSafe"_a = true,
+                             "useWeights"_a = StatisticsControl::WEIGHTS_NONE);
```

Comment:
As you have seen I already added this code. I also modernized the file, which means you should probably just undo your changes. However, I did not wrap the `useWeights` argument because the WeightsBoolean enum was private. Since you have made it public I encourage you to retain this feature (e.g. by adding it to my code).